### ISSUE TITLE
built-in "version"

### ISSUE THREAD

**Issue by fleed** [gsscoder/commandline#433]
I'm using version 2.1.1-beta.

I'm trying to use the built-in verb `version` to display the information, but it's not working. It works if I use the `--version` switch. Am I missing something?

```csharp
var parser = new Parser(settings => settings.HelpWriter = Console.Out);
var parserResult = parser.ParseArguments(args);
```

Is there documentation to use built-in verbs?

**Comment by nemec** [comment]
I don't believe there are any built in verbs anymore with the new `MapResult` style. I'm not sure `version` was ever built in in 1.9.x either. Do the flags not work well for you?
